src/main/java/stirling/software/SPDF/config/security/SecurityConfiguration.java

```diff
@@ -42,6 +54,26 @@ public PasswordEncoder passwordEncoder() {
     @Qualifier("loginEnabled")
     public boolean loginEnabledValue;
 
+    @Autowired
+    @Qualifier("OAUTH2Enabled")
+    public boolean OAUTH2EnabledValue;
+
+    @Autowired
+    @Qualifier("OAUTH2Issuer")
+    public String OAUTH2IssuerValue;
```

Comment:
16:06:44.035 [main] ERROR o.s.b.w.e.tomcat.TomcatStarter - Error starting Tomcat context. Exception: org.springframework.beans.factory.UnsatisfiedDependencyException. Message: Error creating bean with name 'securityConfiguration': Unsatisfied dependency expressed through field 'OAUTH2IssuerValue': No qualifying bean of type 'java.lang.String' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true), @org.springframework.beans.factory.annotation.Qualifier("OAUTH2Issuer")}

These are not nullable should be resolved with other comment about replacing with application properties, but just a fyi that rn it doesnt buld without values setup